cluster/manifests/psp/pod_security_policy.yaml

```diff
@@ -43,3 +43,4 @@ spec:
   - persistentVolumeClaim
   - downwardAPI
   - configMap
+  - projected
```

Comment:
TODO: check which other implications this has

Reply:
[projected volumes](https://kubernetes.io/docs/concepts/storage/volumes/#projected) allow to mount other resources under the same path: configMaps, secrets, downwardAPI and serviceAccount tokens.

Nothing that the user cannot already mount individually, so that should be safe to allow.